src/main/kotlin/frc/robot/autonomous/ScoreState.kt

```diff
@@ -13,13 +16,22 @@ val IS_WITHIN_AUTO_SCORE_DISTANCE = {
     ) <= MAX_ALIGNMENT_DISTANCE
 }
 
-val selectedScorePose: () -> Pose2d = ScorePose.`5L`.pose
+var selectedScorePose: () -> Pose2d = ScorePose.`5L`.pose
 
-// TODO: Add implementation for actual value
-private val selectedHeightPose: () -> ScoreHeight = { ScoreHeight.L3 }
+private var selectedHeightPose: () -> ScoreHeight = { ScoreHeight.L3 }
```

Comment:
Why do you need suppliers? A simple variable will do... just like we have in our subsystems